modules/understanding-agent-install.adoc

```diff
@@ -20,22 +20,22 @@ or
 
 Optional: ZTP manifests
 
-* `cluster-deployment.yaml`
-* `agent-cluster-install.yaml`
-* `pull-secret.yaml`
-* `infraenv.yaml`
-* `cluster-image-set.yaml`
-* `nmstateconfig.yaml`
-* `registries.conf`
-* `ca-bundle.crt`
+* `cluster-manifests/cluster-deployment.yaml`
+* `cluster-manifests/agent-cluster-install.yaml`
+* `cluster-manifests/pull-secret.yaml`
+* `cluster-manifests/infraenv.yaml`
+* `cluster-manifests/cluster-image-set.yaml`
+* `cluster-manifests/nmstateconfig.yaml`
+* `mirror/registries.conf`
+* `mirror/ca-bundle.crt`
 
 [id="agent-based-installer-workflow"]
 == Agent-based Installer workflow
-One of the nodes runs the Assisted Service at the start of the boot process and eventually becomes the bootstrap host. This node is called **node 0**.
```

Comment:
I think we need to mention that the Rendevous host is node 0 because in the diagram we have specified Node 0. it would be confusing to users

Reply:
Yeah, I'd like to update the diagram too, but I don't know where it came from. Did @celebdor draw it? I agree it is potentially confusing if we don't keep the diagram and text in sync.